src/Microsoft.Health.Dicom.SchemaManager/SchemaManagerServiceCollectionBuilder.cs

```diff
@@ -42,7 +41,7 @@ public static IServiceCollection AddSchemaManager(this IServiceCollection servic
         services.AddSingleton<BaseSchemaRunner>();
         services.AddSingleton<IBaseSchemaRunner, DicomBaseSchemaRunner>();
 
-        services.AddMediatR(typeof(SchemaUpgradedNotification).Assembly);
+        services.AddMediatR(c => c.RegisterServicesFromAssembly(typeof(SchemaUpgradedNotification).Assembly));
```

Comment:
I think you can use `RegisterServicesFromAssemblyContaining` instead

Reply:
Changed